.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+# When a new git tag is pushed:
+# * Build and publish a new release to PyPI
+# * Create new GitHub release with placeholder text linking to changelog
+name: Release
+on:
+  push:
+    tags: ['*']
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - name: Build package distributions
+        run: |
+          python -m pip install flit
+          flit build
+      - name: Publish package distributions to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
+        body: "See [changelog](https://github.com/melissawm/sphinx-tags/blob/main/CHANGELOG.md) for release details."
```

Comment:
So to update this link, I would go into the PyPI page for this release and edit the link there, correct?

Reply:
No, that's what will show up under [GitHub releases](https://github.com/melissawm/sphinx-tags/releases), not PyPI. I imagine you will want to manually add a release description, but you can edit the `body` option here to add some useful placeholder text, or a template, or something like that. Or just leave it blank if you prefer.